db/setup/deployment-mass-import.py

```diff
@@ -0,0 +1,528 @@
+def openIndexSystem():
+    """
+    Function to open the system index metadata template file.
+
+    Returns:
+    - An empty system index metadata object.
+    """
+    with open("index-template-system.json", "r") as file:
+        sysData = json.load(file)
+        # print(sysData)
+    return sysData
+
+
+# Function to open the system picture metadata template file
+# Returns an empty system picture metadata object
+def openPictureSystem():
+    """
+    Function to open the system picture metadata template file.
+
+    Returns:
+    - An empty system picture metadata object.
+    """
+    with open("picture-template-system.json", "r") as file:
+        sysData = json.load(file)
+        # print(sysData)
+    return sysData
+
+
+# Create .json from Index data model
+def createJsonIndex(index, name: str, output: str):
+    """
+    Create .json from index data model to the specified output.
+
+    Parameters:
+    - index (dict): The index data model.
+    - name (str): The name of the file.
+    - output (str): The path to the folder.
+
+    Returns:
+    None
+    """
+    data = validator.PIndex(**index)
+    filePath = f"{output}/{name}.json"
+    with open(filePath, "w") as json_file:
+        json.dump(index, json_file, indent=2)
+
+
+# Create .json form Picture data model
+def createJsonPicture(pic, name: str, output: str):
+    """
+    Create .json from picture data model to the specified output.
+
+    Parameters:
+    - pic (dict): The picture data model.
+    - name (str): The name of the file.
+    - output (str): The path to the folder.
+
+    Returns:
+    None
+    """
+    data = validator.PPicture(**pic)
+    extension = name.split(".")[-1]
+    filename = name.removesuffix("." + extension)
+    filePath = f"{output}/{filename}.json"
+    with open(filePath, "w") as json_file:
+        json.dump(pic, json_file, indent=2)
+
+
+# Function to upload the index file located @path to the database
+# RETURNS the indexID of the uploaded index
+def uploadIndexDB(path: str, userID: str):
+    """
+    Upload the index.json file located at the specified path to the database.
+
+    Parameters:
+    - path (str): The path to the index file.
+    - userID (str): The ID of the user.
+
+    Returns:
+    - The ID of the index.
+    """
+
+    # Connect to your PostgreSQL database
+    conn = psycopg.connect(os.getenv("NACHET_DB_URL"))
+
+    # Create a cursor object
+    cur = conn.cursor()
+
+    queries.createSearchPath(conn, cur)
+
+    # Open the file
+    with open(path, "r") as file:
+        data = file.read()
+
+    # Build indexID
+    indexID = uuid.uuid4()
+
+    # Execute the INSERT statement
+    # print(userID)
+    cur.execute(
+        "INSERT INTO indexes (id,index, ownerID) VALUES (%s,%s, %s)",
+        (indexID, data, userID),
+    )
+    conn.commit()
+
+    # Retrieve the index id
+    cur.execute("SELECT id FROM indexes ORDER BY id DESC LIMIT 1")
+
+    cur.close()
+    conn.close()
```

Comment:
Here is a good example of where you would like to use the `wtih` keyword, other wise, you wan to encapsulate the code in a `try`, `except`, `finally` block to be sure the connection to the db will close

Reply:
Noted, I should add some kind of error handling. I expect my procedure to be successful (it should be) and didn't plan error handling in the mass import. This wasn't part of this PR but I've added it!